client/src/lib/fetch/logbook-service.ts

```diff
@@ -101,6 +101,10 @@ const logbookService = {
 		getByUser: async () => api.get<ItemRowsData>({ url: "/data/logbooks/items/rows" }),
 		getByLog: async (log_id: ID) =>
 			api.get<ItemRowsData>({ url: `/data/logbook/log/${log_id}/items/rows` }),
+		getByLogItem: async ({ log_id, item_id }: { log_id: ID; item_id: ID }) =>
```

Comment:
This type `{ log_id: ID, item_id: ID }` is used like 5 times for this one endpoint definition. Should we extract these query type arguments to a file somewhere?